src/components/ModalBackupWords.js

```diff
@@ -244,6 +244,9 @@ class ModalBackupWords extends React.Component {
               {renderWords(6)}
             </tbody>
           </table>
+          <div id="hiddenWordsForTest" style={{ display: 'none' }} >
+            {this.state.words}
+          </div>
```

Comment:
This is a helper for the end-to-end tests, which allows for easy fetching of all the words. Since this is being displayed only when the words are also being displayed visually in plain text to the user, I believe there is no security issue here.